packages/react-app/src/models/Bounty.ts

```diff
@@ -0,0 +1,100 @@
+import mongoose from 'mongoose'
+
+/* Global typing for Bounties */
+export type BountyBoardProps = {
+  _id: any
+  season?: number
+  title: string
+  description: string
+  criteria: string
+  reward: {
+    currency: string
+    amount: number
+  }
+  createdBy: {
+    discordHandle: string
+    discordId: string
+  }
+  claimedBy?: {
+    discordHandle: string
+    discordId: string
+  }
+  createdAt?: string
+  claimedAt?: number
+  completedAt?: number
+  completedBy?: { discordHandle: string; discordId: string }
+  status: string
+  statusHistory?: { status: string; setAt: string }[]
+  submissionUrl?: string
+}
+
+/* BountyBoardSchema will correspond to a collection in your MongoDB database. */
+const BountyBoardSchema = new mongoose.Schema({
+  title: {
+    /* The name of this Bounty */
+
+    type: String,
+    required: [true, 'Please provide a title for this Bounty.'],
+    maxlength: [80, 'Name cannot be more than 60 characters'],
+  },
+  season: {
+    /* The season of this Bounty */
+
+    type: Number,
+    required: [true, 'Bounty Season required'],
+  },
+  description: {
+    /* A short description of the Bounty */
+
+    type: String,
+    required: [true, 'Please provide the bounty description'],
+  },
+  criteria: {
+    /* Acceptance criteria of deliverables for the Bounty to be marked complete. */
+
+    type: String,
+    required: [true, 'Please provide the bounty acceptance criteria'],
+  },
+  reward: {
+    /* Bounty reward */
+
+    currency: String,
+    amount: Number,
+    type: Object,
+    required: [true, 'Please provide the bounty reward'],
+  },
+  createdBy: {
+    /* Discord identity of bounty creator */
+
+    discordHandle: String,
+    discordId: Number,
+    type: Object,
+    required: [true, 'Please provide your discord information'],
+  },
+  createdAt: {
+    /* Date of Bounty creation */
+
+    type: Date,
```

Comment:
This might need to be store as a string as the schema expects ISO 8601 as a **string** since we agreed early on in that format.

Also if you guys 
@Oktalize @Behold3th have some time maybe we can go over the mongo db insertion. I would really like to get the validation schema in place for MVP?

Reply:
New commit, I changed this to `String`.